mlos_bench/mlos_bench/tests/optimizers/conftest.py

```diff
@@ -102,6 +102,11 @@ def flaml_opt_max(tunable_groups: TunableGroups) -> MlosCoreOptimizer:
         },
     )
 
+# FIXME: SMAC's RF model can be non-deterministic at low iterations, which are
+# normally calculated as a percentage of the max_iterations and number of
+# tunable dimensions, so for now we set the initial random samples equal to the
+# number of iterations and control them with a seed.
```

Comment:
@motus FYI ^^^

Reply:
*argh!* SMAC is override that:
https://github.com/automl/SMAC3/blob/541ee7e0383b491b86d1a23dcff669f2efad616d/smac/initial_design/abstract_initial_design.py#L92

Reply:
Added a way to pass the override for that too: `max_ratio`

This feels kludgey, but I don't want to be tracking down random seed issues anymore and just want the tests to pass reliably for now.

I filed a bug (#482) so we can come back and address this in the future.  Am worried about reproducible science mostly, but that will come with lots of data and we need to move on to get to that.